portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
```

Comment:
Unless I'm misunderstanding  the order I should review these in (which totally might be the case!), It seems like these greetings are not being used in the PR. 

I think it might be better to make the change related to this function in a different PR, so the way it is being called is grouped together.

Reply:
Yes that function is not being used, I remove the functions in later PRs, will that affect anything?

Reply:
Hey totally cool to remove the function in a later PR. But this PR is also including commit `582308c` which makes changes to the content of the function, which leads the reviewers to believe that it is something to consider when making the review.

I think it might make sense to `git revert 582308c` ( not sure on exact terminology) on this branch, so that the PR is more focused on the rest of the changes. :)

Reply:
Alternatively, you could add another commit to this PR that removes this function and the file it's in entirely. Since you removed the one usage of the function, it doesn't make sense to keep it around. 

This is one example of wanting to group related changes together in a PR. If you remove all usages of a function in a PR, you should also remove that function in the same PR, so it doesn't stick around unused.